client/my-sites/email/email-providers-comparison/style.scss

```diff
@@ -202,3 +200,51 @@
 		}
 	}
 }
+
+.email-providers-comparison__gsuite-user-list-actions-container {
+	$container-top-margin: 1em;
+	$container-left-margin: 1em;
+	$child-top-margin: 1em;
+	$child-left-margin: 1em;
+
+	display: flex;
+	flex-direction: column;
+	margin-top: $container-top-margin;
+
+	@include breakpoint-deprecated( '>480px' ) {
+		flex-direction: row;
+		margin-top: 0;
+		margin-left: $container-left-margin;
+	}
+
+	@include breakpoint-deprecated( '>660px' ) {
+		flex-direction: column;
+		margin-left: 0;
+		margin-top: 0;
+	}
+
+	@include breakpoint-deprecated( '>800px' ) {
+		flex-direction: row;
+		margin-top: 0;
+		margin-left: $container-left-margin;
+	}
+
+	> * + * {
+		margin-top: $child-top-margin;
+
+		@include breakpoint-deprecated( '>480px' ) {
+			margin-top: 0;
+			margin-left: $child-left-margin;
+		}
+
+		@include breakpoint-deprecated( '>660px' ) {
+			margin-left: 0;
+			margin-top: $child-top-margin;
+		}
+
+		@include breakpoint-deprecated( '>800px' ) {
```

Comment:
For all these conditions that are using `breakpoint-deprecated()`, is there some other CSS that we're overriding that we need so many breakpoint-specific styles? I think a comment to describe why we need this would be really useful to future readers, as having these flip-flopping styles feels really weird. (I would also appreciate having the  items in the same order so it's easier to see how each modification is occurring.)

Reply:
Good point and thanks! I've added a comment about the need and fixed the order of property declarations.